R/test.data.table.R

```diff
@@ -366,12 +366,20 @@ test = function(num,x,y=TRUE,error=NULL,warning=NULL,output=NULL,message=NULL) {
     # For test 617 on r-prerel-solaris-sparc on 7 Mar 2013
     # nocov start
     if (!fail) {
-      cat("Test",numStr,"ran without errors but failed check that x equals y:\n")
-      cat("> x =",deparse(xsub),"\n")
-      if (is.data.table(x)) compactprint(x) else {cat("First 6 of ", length(x)," (type '", typeof(x), "'): ", sep=""); print(head(x))}
```

Comment:
was seeing this in the output

![Screenshot 2019-07-08 at 5 13 59 PM](https://user-images.githubusercontent.com/7606389/60803800-981e1300-a1ae-11e9-96ab-769c75c21c5b.png)

so originally came here to add the '\n' & stayed to clean things up a bit.

Reply:
could we isolate changes related to `test` into new PR?